agent/06_agent_create_cluster.sh

```diff
@@ -68,6 +68,11 @@ function create_factory_image() {
 
 function create_config_image() {
 
+    # Copy any extra manifests
+    if [ -d "${asset_dir}/openshift" ]; then
```

Comment:
It's not likely something subjected to change frequently, but just for the sake of clarity - and to avoid issues if amended in future, I'd suggest to export `$EXTRA_MANIFESTS_PATH` in `common.sh` and use it here.

Reply:
Updated to use common EXTRA_MANIFESTS_PATH